src/mapper/dynakubes.go

```diff
@@ -101,6 +101,10 @@ func (dm DynakubeMapper) updateNamespaces(modifiedNs []*corev1.Namespace) error
 		if err := dm.client.Update(dm.ctx, ns); err != nil {
 			return err
 		}
+		log.Info(
```

Comment:
this leads to spam in case there are maaany namespaces, that is why we didn't have it

Reply:
I was puzzled by fictional messages _started monitoring namespace_. Noticed some which are solely intentional and ignored. The mutated status helped me to focus on the factual.

Is not the _updated namespace_ valuable?
```
{"ts":"07:45:27.093Z",
    "msg":"checking namespace labels not necessary","namespace":"default"}
{"ts":"07:45:27.093Z",
    "msg":"namespace request","namespace":"default","operation":"UPDATE"}
{"ts":"07:45:27.106Z",
    "msg":"checking namespace labels not necessary","namespace":"dt-synthetic"}
{"ts":"07:45:27.106Z",
    "msg":"namespace request","namespace":"dt-synthetic","operation":"UPDATE"}
{"ts":"07:48:03.232Z",
    "msg":"started monitoring namespace","namespace":"default",}
{"ts":"07:48:03.232Z",
    "msg":"started monitoring namespace","namespace":"dynatrace",}
{"ts":"07:48:04.123Z",
    "msg":"started monitoring namespace","namespace":"default",}
{"ts":"07:48:04.123Z",
    "msg":"started monitoring namespace","namespace":"dynatrace",}
{"ts":"07:48:04.132Z",
    "msg":"checking namespace labels not necessary","namespace":"default"}
{"ts":"07:48:04.132Z",
    "msg":"namespace request","namespace":"default","operation":"UPDATE"}
{"ts":"07:48:04.149Z",
    "msg":"updated namespace","namespace":"default",}
{"ts":"07:48:04.159Z",
    "msg":"updated namespace","namespace":"dynatrace",}
```
[neglected-namespace-labeling.log](https://github.com/Dynatrace/dynatrace-operator/files/10111054/neglected-namespace-labeling.log)

Reply:
we used to have it, but it was spam.
the "started monitoring" happens for the first time a namespace was labeled